addons/medical_damage/stringtable.xml

```diff
@@ -1,78 +1,18 @@
-                <Czech>Jaké poškození může hráč dostat než bude zabit?</Czech>
-                <Portuguese>Qal é o dano que um jogador pode sofrer antes de morrer?</Portuguese>
-                <French>Quels dégâts peut subir un joueur avant d'être tué</French>
-                <Hungarian>Mennyi sérülést szenvedhet el egy játékos, mielőtt meghal?</Hungarian>
-                <Italian>Quanto è il danno che un giocatore può sostenere prima di essere ucciso?</Italian>
-                <Japanese>プレイヤーが死に始める前に損傷を受けるようにしますか?</Japanese>
-                <Korean>얼마정도의 부상을 플레이어가 죽기 전까지 버틸 수 있습니까?</Korean>
-                <Chinesesimp>玩家死亡前所能承受的伤害程度</Chinesesimp>
-                <Chinese>玩家死亡前所能承受的傷害程度</Chinese>
-            </Key>
-            <Key ID="STR_ACE_Medical_damage_AIDamageThreshold_DisplayName">
-                <English>AI Damage</English>
-                <Russian>Урон ботам</Russian>
-                <Polish>Próg obrażeń AI</Polish>
-                <Spanish>Daño IA</Spanish>
-                <German>KI-Schaden</German>
-                <Czech>Poškození AI</Czech>
-                <Portuguese>Dano da IA</Portuguese>
-                <French>Dégâts des IA</French>
-                <Hungarian>AI sérülés</Hungarian>
-                <Italian>Danno AI</Italian>
-                <Japanese>AI への損傷</Japanese>
-                <Korean>인공지능 부상</Korean>
-                <Chinesesimp>AI伤害</Chinesesimp>
-                <Chinese>AI傷害</Chinese>
-            </Key>
-            <Key ID="STR_ACE_Medical_damage_AIDamageThreshold_Description">
-                <English>What is the damage an AI can take before being killed?</English>
-                <Russian>Какой уровень урона необходим, чтобы убить бота?</Russian>
-                <Polish>Jaki jest próg obrażeń, jakie AI może otrzymać zanim zostanie zabite?</Polish>
-                <Spanish>¿Cuál es el daño que la IA puede sufrir antes de morir?</Spanish>
-                <German>Wie viel Schaden kann eine KI erleiden, bis sie getötet wird?</German>
-                <Czech>Jaké poškození může AI dostat než bude zabito?</Czech>
-                <Portuguese>Qual é o dano que uma IA pode sofrer antes de morrer?</Portuguese>
-                <French>Quels dégâts peut subir une IA avant d'être tuée</French>
-                <Hungarian>Mennyi sérülést szenvedhet el egy AI, mielőtt meghal?</Hungarian>
-                <Italian>Quanto è il danno che un'IA può sostenere prima di essere uccisa?</Italian>
-                <Japanese>AI が死に始める前に損傷を受けるようにしますか?</Japanese>
-                <Korean>얼마정도의 부상을 인공지능이 죽기 전까지 버틸 수 있습니까?</Korean>
-                <Chinesesimp>AI 死亡前所能承受的伤害程度</Chinesesimp>
-                <Chinese>AI 死亡前所能承受的傷害程度</Chinese>
-            </Key>
-        </Container>
+        <Key ID="STR_ACE_Medical_Damage_PlayerDamageThreshold_DisplayName">
+            <English>Player Damage Threshold</English>
```

Comment:
Would be tempted to rename these damage settings as they look more like a health coefficient than a damage threshold.

Reply:
Believe in meeting we discussed something like "Critical Damage Threshold" > "Damage event which will cause a unit to fall unconscious" (reword as desired).